patches/framework-bundle-lock-env-with-service.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/DependencyInjection/FrameworkExtension.php b/DependencyInjection/FrameworkExtension.php
+index 66af2771b..56de4975a 100644
+--- a/DependencyInjection/FrameworkExtension.php
++++ b/DependencyInjection/FrameworkExtension.php
+@@ -2207,6 +2207,7 @@ class FrameworkExtension extends Extension
+                     $resourceStore = 'null';
+                 }
+ 
++                $usedEnvs = [];
```

Comment:
we cannot have patches in shopware

Reply:
Its just temporary for testing until patch merged. Will remove when its ready.